packages/flutter/lib/src/cupertino/search_field.dart

```diff
@@ -140,6 +140,7 @@ class CupertinoSearchTextField extends StatefulWidget {
     this.decoration,
     this.backgroundColor,
     this.borderRadius,
+    this.keyboardType,
```

Comment:
Can you set the default of `TextInputType.text` here?  I noticed that you mentioned the default in your docs below.  I think it's slightly misleading if the default is only set in the CupertinoTextField but not CupertinoSearchTextField.